x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_dashboards_view/content_pack_objects_list.tsx

```diff
@@ -6,7 +6,7 @@
  */
 import { EuiBadge, EuiBasicTable } from '@elastic/eui';
 import React from 'react';
-import { DashboardAttributes } from '@kbn/dashboard-plugin/common/content_management/v2';
+import type { DashboardSavedObjectAttributes } from '@kbn/dashboard-plugin/server';
```

Comment:
The `DashboardAttributes` type is distinct from the `DashboardSavedObjectAttributes` type which you are currently using. So we need to ensure the type is correct for this usage.